webapp/src/reducers.ts

```diff
@@ -34,16 +32,25 @@ import {
     RECEIVED_CALLS_CONFIG,
     SHOW_END_CALL_MODAL,
     HIDE_END_CALL_MODAL,
+    RECEIVED_CHANNEL_STATE,
 } from './action_types';
 
-const isVoiceEnabled = (state = false, action: { type: string }) => {
+interface channelState {
+    id: string,
+    enabled: boolean,
+}
```

Comment:
I wonder why we're making this a channelState now, is it in anticipation for future details?
Because all we really need to store is `enabled` since the channelID is the index...

Reply:
I am just trying to bring back some consistency with the server-side API and also start moving into the direction of receiving the whole state at once to avoid races. At some point this could hold the call state as well. Of course we are still planning a proper refactor around all of this.

As far as the ID, it's totally redundant but in my experience it's worth having it because the index is metadata that could go away as soon as we change how the actual data is stored but the data/type has an id. If we were worried about memory I would remove it but I don't think that's a concern at this time.